sys/vane/ames.hoon

```diff
@@ -1665,9 +1354,46 @@
     ^-  [(list move) _+>]
     ?-  +<.sih
         %crud  [[[hen [%slip %d %flog +.sih]] ~] +>]
+    ::
         %mack  ?~  +>.sih  $(sih [%g %nice ~])          ::  XX using old code
                $(sih [%g %mean `[%mack +>+.sih]])
+    ::
+        %turf
+      =.  tuf.fox  turf.sih
+      ::  XX give %turf if already recvd %bar?
```

Comment:
Terrible comment, made worse by a typo ;) There's an abstract race condition on first boot here. (There's actually lots of these across vane initialization). But it's only abstract, because of a workaround/hack in vere. I'll remove the comment.

Reply:
Indeed. Correcting (or at least clarifying) this is one of the highest priorities after integrating cc-release, imo. Before that, I'm not sure it's worth pursuing. At this time, I don't know really know more than what I've already said here ...